_posts/2016/2016-12-15-hr-ui.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Working as a UI-Designer at ePages"
+date: "2016-12-15 12:11:00"
+image: blog-header/ui-designer.jpg
+categories: working-at-epages
+authors: ["Rose, Serena", "Verena"]
+---
+
+So far, we have interviewed a [Ruby Developer](https://developer.epages.com/blog/2016/02/04/hr-ruby.html), a [Java Developer](https://developer.epages.com/blog/2016/03/01/hr-java.html), a [QA engineer](https://developer.epages.com/blog/2016/04/12/hr-quality-assurance.html) and a [Software Developer](https://developer.epages.com/blog/2016/06/01/hr-backend.html) to learn about working at ePages.
+For this post, we asked a UI-Designer in Hamburg about her daily work.
+
+## Here's her story:
+
+Hi, my name is Verena and I work as a User Interface Designer at ePages since July 2012.
+I originally started as an intern in January 2012 in Jena, but then got offered to work as a full-time employee, which I happily accepted.
+When I started in Jena, I was involved in a project based in Jena and Hamburg, simultaneously.
+In July 2015, the project was finally relocated to Hamburg, and I was asked to move along with the project.
+After some thinking, I took the opportunity and I now have been living in Hamburg for already a year and four months.
+The project's name is Unity and we are developing a new shop software using frontend and backend technology.
+Currently, a beta version is on the market.
+We hope the new shop design, which includes simpler functionalities as well as a responsive storefront will be accepted and approved by the users.
+
+## How did you get to know ePages?
+
+Originally, I studied graphic design.
+I, then, decided that I wanted to head more into an e-commerce direction and work with programs such as HTML, LESS, JavaScript, and alike.
```

Comment:
maybe "tools" rather than languages or programs? but it's up to @vzerwas to decide that.